src/Lime.Client.TestConsole/Properties/AssemblyInfo.cs

```diff
@@ -51,5 +51,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.7.0.0")]
-[assembly: AssemblyFileVersion("0.7.0.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
The current version on Nuget is "0.10.x.x", let's use that here to make this coherent.

Reply:
done